frontend/src/app/Header.tsx

```diff
@@ -19,8 +20,7 @@ const Header: React.FC<HeaderProps> = ({ onNotificationsClick }) => {
           alt={`${ODH_PRODUCT_NAME} Logo`}
         />
       }
-      logoComponent={Link}
-      logoProps={{ to: '/' }}
+      logoProps={{ onClick: () => navigate('/') }}
```

Comment:
Why this change?

Reply:
This is because there is a type error when we try to pass Link as the logoComponent
```
Type 'ForwardRefExoticComponent<LinkProps & RefAttributes<HTMLAnchorElement>>' is not assignable to type 'ReactNode'.
```